client/src/services/nftServices.js

```diff
@@ -65,13 +67,26 @@ export const getTransferDeploy = ({ publicKey, recipient, nftContract, tokenId }
 		const recipientPK = CLPublicKey.fromHex(recipient);
 		const contractHashByteArray = contractHashToByteArray(nftContract);
 		const pbKey = CLPublicKey.fromHex(publicKey);
-
-		const runtimeArgs = RuntimeArgs.fromMap({
+		let mapping = {
 			sender: createRecipientAddress(pbKey),
 			recipient: createRecipientAddress(recipientPK),
-			token_id: CLValueBuilder.string(tokenId),
-		});
-		return getTransferNFTDeploy(pbKey, runtimeArgs, contractHashByteArray, toMotes(2.5));
+		};
+
+		const oldNFTAddresses = getConfigKey('OLD_NFT_SMART_CONTRACT_ADDRESSES')
```

Comment:
I will test the tesnet integration after this PR (https://github.com/CasperDash/casperdash-api/pull/31) merged